moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -35,5 +35,5 @@ add_executable(moveit_list_request_adapter_plugins src/list.cpp)
 target_link_libraries(moveit_list_request_adapter_plugins ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
 install(TARGETS ${MOVEIT_LIB_NAME} moveit_list_request_adapter_plugins
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
\ No newline at end of file
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
these kind of indentation changes should be avoided. it looks like nothing actually changed in this file. is there a standard you are following in making this change?

Reply:
I'd put it more bluntly: a hotfix PR shouldn't have whitespace changes. 

If you think this is a general issue we should do it for the whole repo at once.

Reply:
I just reverted unnecessary whitespace changes of the original PR. 
Interesting that you complain here, but not on the original PR.

Reply:
i didn't bring it up in the original PR because 1) there were actual changes to the CMakeList beyond just formatting and 2) managing GSoC students is hard and you pick your battles :)